qa_forum/models.py

```diff
@@ -0,0 +1,24 @@
+from django.contrib.auth.models import User
```

Comment:
I would use a model folder if there are too many models. For here it should be acceptable.